File: fixtures/scale/dart/large/lib/registry/transposition_registry.dart
import '../core/cipher_suite.dart';
import '../ciphers/railfence/railfence_suite.dart';
import '../ciphers/columnar/columnar_suite.dart';
import '../ciphers/scytale/scytale_suite.dart';
import '../ciphers/route/route_suite.dart';
import '../ciphers/myszkowski/myszkowski_suite.dart';

List<CipherSuite> transpositionSuites() => [
  RailFenceSuite(),
  ColumnarSuite(),
  ScytaleSuite(),
  RouteSuite(),
  MyszkowskiSuite(),
];

File: fixtures/scale/kotlin/large/src/cryptobench/registry/TranspositionRegistry.kt
package cryptobench.registry

import cryptobench.core.CipherSuite
import cryptobench.ciphers.railfence.RailFenceSuite
import cryptobench.ciphers.columnar.ColumnarSuite
import cryptobench.ciphers.scytale.ScytaleSuite
import cryptobench.ciphers.route.RouteSuite
import cryptobench.ciphers.myszkowski.MyszkowskiSuite

object TranspositionRegistry {
    fun suites(): List<CipherSuite> = listOf(
        RailFenceSuite(),
        ColumnarSuite(),
        ScytaleSuite(),
        RouteSuite(),
        MyszkowskiSuite(),
    )
}
